RunningOrder/TutoView.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  TutoView.swift
+//  RunningOrder
+//
+//  Created by Clément Nonn on 22/09/2020.
+//  Copyright © 2020 Worldline. All rights reserved.
+//
+
+import SwiftUI
+
+struct TutoView: View {
+
+    @Binding var space: Space?
+    @State private var newSpaceName = ""
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: 20) {
+            Text("Welcome").font(.largeTitle)
+            Text("You don't have yet your space, or joined a shared space")
```

Comment:
do we want to localise the texts?

Reply:
yes, but the keys are the English text anyway :)

Reply:
of course :)
does the localised string exist?

Reply:
not yet